test/e2e/tests/edit-gas-fee.spec.js

```diff
@@ -238,17 +241,17 @@ describe('Editing Confirm Transaction', function () {
           '.currency-display-component__text',
         );
         const transactionAmount = transactionAmounts[0];
-        assert.equal(await transactionAmount.getText(), '3');
+        assert.equal(await transactionAmount.getText(), '0');
```

Comment:
Did we lower the amount that the send transaction button sends?

Reply:
we did

>Updated edit-gas-fee.spec.js as the Send flow on the test-dapp was updated, reducing the ETH amount to send from 3 to 0 https://github.com/MetaMask/test-dapp/pull/129